internal/perses/common/perses_client_factory.go

```diff
@@ -41,8 +54,13 @@ func NewWithURL(url string) PersesClientFactory {
 }
 
 func (f *PersesClientFactoryWithURL) CreateClient(config persesv1alpha1.Perses) (v1.ClientInterface, error) {
```

Comment:
Since this method doesn't use the config I guess we can remove it

Reply:
Sure! Removed in 9847bb16fe454de599ab37c596cfe4d4ede6ee30 commit :)